Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -197,89 +198,123 @@ void GeneralWidget::AddDescriptions()
 // We need QObject::tr
 #if defined(_WIN32)
   static const char TR_BACKEND_DESCRIPTION[] = QT_TR_NOOP(
-      "Selects which graphics API to use internally.\n\nThe software renderer is extremely "
+      "Selects which graphics API to use internally.<br><br>The software renderer is extremely "
       "slow and only useful for debugging, so either OpenGL, Direct3D, or Vulkan are "
       "recommended. Different games and different GPUs will behave differently on each "
       "backend, so for the best emulation experience it is recommended to try each and "
-      "select the backend that is least problematic.\n\nIf unsure, select OpenGL.");
+      "select the backend that is least problematic.<br><br><dolphin_emphasis>If unsure, "
+      "select OpenGL.</dolphin_emphasis>");
 #else
   static const char TR_BACKEND_DESCRIPTION[] = QT_TR_NOOP(
-      "Selects which graphics API to use internally.\n\nThe software renderer is extremely "
+      "Selects which graphics API to use internally.<br><br>The software renderer is extremely "
       "slow and only useful for debugging, so any of the other backends are "
-      "recommended.\n\nIf unsure, select OpenGL.");
+      "recommended.<br><br><dolphin_emphasis>If unsure, select OpenGL.</dolphin_emphasis>");
 #endif
   static const char TR_ADAPTER_DESCRIPTION[] =
-      QT_TR_NOOP("Selects a hardware adapter to use.\n\nIf unsure, select the first one.");
+      QT_TR_NOOP("Selects a hardware adapter to use.<br><br><dolphin_emphasis>If unsure, "
+                 "select the first one.</dolphin_emphasis>");
   static const char TR_FULLSCREEN_DESCRIPTION[] =
-      QT_TR_NOOP("Uses the entire screen for rendering.\n\nIf disabled, a "
-                 "render window will be created instead.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Uses the entire screen for rendering.<br><br>If disabled, a "
+                 "render window will be created instead.<br><br><dolphin_emphasis>If "
+                 "unsure, leave this unchecked.</dolphin_emphasis>");
   static const char TR_AUTOSIZE_DESCRIPTION[] =
-      QT_TR_NOOP("Automatically adjusts the window size to the internal resolution.\n\nIf unsure, "
-                 "leave this unchecked.");
-
+      QT_TR_NOOP("Automatically adjusts the window size to the internal resolution.<br><br "
```

Comment:
You missed one `<br />` ;)

Hopefully it will be wrapped in a better way with `<br>`.